phpBB/includes/acp/acp_forums.php

```diff
@@ -1960,7 +1960,7 @@ function delete_forum_content($forum_id)
 		$row = $db->sql_fetchrow($result);
 		$db->sql_freeresult($result);
 
-		set_config('num_posts', (int) $row['stat'], true);
+		$config->set('num_posts', (int) $row['stat'], false);
```

Comment:
The parameter is not used anyway (as far as I could see). I don't know if it was something planned or removed.